codegen/src/main/java/software/amazon/awssdk/codegen/model/intermediate/AcceptorModel.java

```diff
@@ -64,7 +61,14 @@ public void setMatcher(String matcher) {
     }
 
     public String getArgument() {
-        return argument;
+        String[] unformatted = argument.split("\\.");
+        StringJoiner formatted = new StringJoiner(".");
+
+        for (String format : unformatted) {
+            formatted.add(StringUtils.uncapitalize(format));
```

Comment:
We fixed this in v2 with better and more consistent model naming. dbInstance in v2 vs dBInstance in v1